examples/example-gateway/config/test.json

```diff
@@ -45,6 +45,7 @@
 	"clients.withexceptions.timeout": 10000,
 	"clients.withexceptions.maxConcurrentRequests": 1000,
 	"clients.withexceptions.errorPercentThreshold": 20,
+	"contextlogger.skipzanzibarlogs": false,
```

Comment:
Nit: Do we need to mention `zanzibarLogs`  again?

Suggestion `contextlogger.zanzibar.skip: true` is more readable and easily extensible.

 For example, I can say the following for future use cases
```
contextlogger.Y.skip : true
contextlogger.X.skip : false
contextlogger.{{""}}.skip : false

``` 

It also makes it easier to search for `contextLogger.(.*).skip` in the regex.

Reply:
contextlogger.zanzibar.skip does not make full sense, made it contextlogger.zanzibar.skiplogs